supertokens_python/recipe/emailverification/interfaces.py

```diff
@@ -236,11 +236,14 @@ def __init__(self, id: str, email: str):
 
 class TypeEmailVerificationEmailDeliveryInput:
     def __init__(
-        self, user: TypeEmailVerificationEmailDeliveryInputUser,
-        email_verify_link: str, user_context: Dict[str, Any]
+        self,
+        user: TypeEmailVerificationEmailDeliveryInputUser,
+        email_verify_link: str,
+        user_context: Dict[str, Any]
```

Comment:
send_email function takes email_input and user_context as parameter. I think you can reomve user_context from here